src/styles/abstracts/_variables.scss

```diff
@@ -5,6 +5,7 @@ $background: #c0c0c0;
 $light: #fff;
 $dark: #000;
 $shuttle-gray: #5A6570;
+$blue: #47a1f5;
```

Comment:
Please use color namer to name this color.
Source: https://chir.ag/projects/name-that-color/#6195ED

Reply:
Very interesting 🧐. I'll rename it then.